src/pages/NewChatSelectorPage.tsx

```diff
@@ -1,51 +1,38 @@
-import {useNavigation} from '@react-navigation/native';
 import React from 'react';
-import {withOnyx} from 'react-native-onyx';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import ScreenWrapper from '@components/ScreenWrapper';
 import TabSelector from '@components/TabSelector/TabSelector';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
-import withWindowDimensions, {windowDimensionsPropTypes} from '@components/withWindowDimensions';
-import compose from '@libs/compose';
 import OnyxTabNavigator, {TopTab} from '@libs/Navigation/OnyxTabNavigator';
 import CONST from '@src/CONST';
-import ONYXKEYS from '@src/ONYXKEYS';
 import NewChatPage from './NewChatPage';
 import WorkspaceNewRoomPage from './workspace/WorkspaceNewRoomPage';
+import useLocalize from '@hooks/useLocalize';
 
-const propTypes = {
-    ...windowDimensionsPropTypes,
-
-    ...withLocalizePropTypes,
-};
-
-const defaultProps = {
-    betas: [],
-    personalDetails: {},
-    reports: {},
-};
-
-function NewChatSelectorPage(props) {
-    const navigation = useNavigation();
 
+function NewChatSelectorPage() {
+    const {translate} = useLocalize();
     return (
-        <ScreenWrapper
+        <ScreenWrapper 
             shouldEnableKeyboardAvoidingView={false}
             includeSafeAreaPaddingBottom={false}
             shouldShowOfflineIndicator={false}
             shouldEnableMaxHeight
             testID={NewChatSelectorPage.displayName}
         >
-            <HeaderWithBackButton
-                title={props.translate('sidebarScreen.fabNewChat')}
-                onBackButtonPress={navigation.goBack}
-            />
+            <HeaderWithBackButton title={translate('sidebarScreen.fabNewChat')} />
             <OnyxTabNavigator
                 id={CONST.TAB.NEW_CHAT_TAB_ID}
-                tabBar={TabSelector}
+                tabBar={({state, navigation, position}) => (
+                    <TabSelector
+                        state={state}
+                        navigation={navigation}
+                        position={position}
+                    />
+                )}
             >
                 <TopTab.Screen
                     name={CONST.TAB.NEW_CHAT}
+                    // @ts-expect-error TODO: 'isGroupChat' is declared here.
```

Comment:
Let's remove this ts expect error, instead let's make `isGroupChat` optional

Reply:
Ok.